code/game/objects/structures/window_frame.dm

```diff
@@ -13,7 +13,12 @@
 	var/basestate = "window"
 	var/junction = 0
 	var/reinforced = FALSE
-
+	var/damage = 0
```

Comment:
Same as #189 this should be on /structure. Feel free to combine these 2 prs if you fix it up so structures handle it properly.